### ISSUE TITLE
[Insomnia Designer] Windows : How can I copy server response

### ISSUE THREAD

**Issue by Jean-MarcCheynier** [getinsomnia/insomnia#2278]
**How to copy Response body**

In insomnia designer when I execute a request using the preview on the right sidee, I can see the response from the server in a fancy black square, but I cannot copy this response. **I can only click on the "Download" button** and download it in a separate file... I find it not practical did you prevent the text selection on purpose ?

Insomnia designer, windows 10.
![Capture](https://user-images.githubusercontent.com/22655382/84320956-bf295580-ab72-11ea-8521-f6d11025acdb.PNG)

**Comment by sonicyeti** [comment]
On it, thanks @Jean-MarcCheynier!
